services/scan/src/precinct_scanner_app.ts

```diff
@@ -459,7 +459,10 @@ export async function buildPrecinctScannerApp(
     }
   );
 
-  app.get('/precinct-scanner/backup', (_request, response) => {
+  app.get('/precinct-scanner/backup', (request, response) => {
+    const scanImagesToInclude = parseScanImagesToIncludeQueryParam(
+      request.query['scanImagesToInclude']
+    );
```

Comment:
Any particular reason to have a query parameter and determine this on the frontend vs just doing it all on the backend?

Reply:
Mm a really great question - thinking about this some more, looking at the sheet counts on the server would be a really clean and simple way to do this

I'm going to merge this PR as is to have a mitigation in place and to unblock others, but I'll explore the alternative and potentially get a PR up for the alternative tomorrow morning

Reply:
@jonahkagan I did revisit this and tweaked the implementation in https://github.com/votingworks/vxsuite/pull/2484!